projector.sh

```diff
@@ -4,20 +4,34 @@ PROJECTOR_BIN_PATH='projector_bin'
 projector_prompt() {
   local projector_path="$(projector_path_find $PWD)"
 
-  test -z $projector_path && return 0
-
   projector_path_changed "$projector_path" || \
     projector_source_modified "$projector_path" && \
     projector_source_bash_project "$projector_path"
 
-  set_path_with_projector_path "$projector_path" || return 0
+  projector_path_changed "$projector_path" && \
+    set_path_with_projector_path "$projector_path"
+
+  projector_prompt_post $projector_path
+}
+
+projector_prompt_post() {
+  local file="$1/$PROJECTOR_SOURCE_PATH"
+
+  PROJECT_HOME="$1"
```

Comment:
This doesn't seem to be used, what's the intent of this?

As a side note, are there any other environment variables that should be documented?

Reply:
`PROJECT_HOME` this environment variable is used to detect if changed to a different projector directory.

It can also be used in your own functions or scripts to make sure the directory things are run from are relative to the projector home.